tutorbackup/templates/backup/build/backup/restore_services.py

```diff
@@ -175,7 +203,9 @@ def download_from_s3(version_id=None):
 @click.option('--version', default="", type=str,
               help="Version ID of the backup file")
 @click.option('--download', is_flag=True, help="Download from S3")
-def main(exclude, version, download):
+@click.option('--list-versions', is_flag=True,
```

Comment:
Rather than make it a flag and hardcode 20, you could just accept an argument for the number of versions listed, no?

Reply:
will do